.gitignore

```diff
@@ -1,2 +1,7 @@
 .vscode/*
-.DS_Store
\ No newline at end of file
+.DS_Store
+debug.log
+css/*
+scss/*.css
+scss/*.css.map
```

Comment:
Yeah I do think we should keep the css folder, just remove the css files from scss folder manually.